replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/data/ReplicatorConfigImpl.java

```diff
@@ -41,9 +42,19 @@
 
   private static final String SUSPENDED_KEY = "suspended";
 
+  private static final String MODIFIED_KEY = "modified";
+
   /**
-   * 0/No Version - initial version of configs which were saved in the catalog framework. 1 - the
-   * first version of configs to be saved in the replication persistent store.
+   * Field specifying the version of the configuration. Possible versions include:
```

Comment:
❓ We can get rid of this section right? I think since we will be resetting back to version 1 in our new deployment, we won't have to worry about compatibility concerns with 0.2.x?

Reply:
This change needs to go in 0.2.x and I am not sure we will be able to ignore versioning from 0.2.x since in the future, we will be communicating with old replication-based nodes too. BTW, do we expose the version as part of the GraphQL API? If not, will that cause us problems when we try to communicate with a node or are we going to have to hardcode that a given old-replication-based version used a given set of versions for its objects? That might become messy when it is upgraded. 🤔

Reply:
No we don't expose it through GraphQL right now.

Reply:
We might want to consider exposing it